docs/supported_apis/utilities_supported.rst

```diff
@@ -11,11 +11,14 @@ The second column is a flag for whether or not there is an implementation in Mod
 the method in the left column. ``Y`` stands for yes, ``N`` stands for no, ``P`` stands
 for partial (meaning some parameters may not be supported yet), and ``D`` stands for
 default to pandas.
+**Currently second column reflects implementation status for Ray and Dask engines. By default support of a method
+in the Omnisci engine could be treated as ``D`` unless ``Notes`` column contains additional information. imilarly,
+by default ``Notes`` contains information about ``Ray`` and ``Dask`` engines unless ``Omnisci`` explicitly mentioned.**
 
 +---------------------------+---------------------------------+----------------------------------------------------+
 | Utility method            | Modin Implementation? (Y/N/P/D) | Notes for Current implementation                   |
 +---------------------------+---------------------------------+----------------------------------------------------+
-| `pd.concat`_              | Y                               |                                                    |
+| `pd.concat`_              | Y                               |  **Omnisci**:``P``?                                |

```

Comment:
same as for `append`: sort and ignore_index parameters are ignored in certain cases